features/shipping/managing_shipping_methods/browsing_shipping_methods.feature

```diff
@@ -0,0 +1,19 @@
+@managing_shipping_methods
+Feature: Browsing shipping methods
+    In order to have a overview of all defined shipping methods
+    As an Administrator
+    I want to be able to browse list of them
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store allows shipping with "UPS Carrier"
+        And the store allows shipping with "FedEx Carrier"
+        And the store allows shipping with "DHL Carrier"
+        And the store allows shipping with "TNT Carrier"
```

Comment:
Wdyt about merging these steps in sth like ``And the store allows shipping with "UPS Carrier", "FedEx Carrier", "DHL Carrier" and "TNT Carrier"``?